docker/README.md

```diff
@@ -0,0 +1,149 @@
+# Building KubeStellar deployment as a container
+
+Table of contents:
+- [KubeStellar in a container](#kubestellar-in-a-container)
+  - [Install the pre-requisites](#install-the-pre-requisites)
+  - [Build a local container image](#build-a-local-container-image)
+  - [Build and push a multi-architecture image](#build-and-push-a-multi-architecture-image)
+  - [Run the container and access KubeStellar from the host OS](#run-the-container-and-access-kubestellar-from-the-host-os)
+  - [Checking the container logs](#checking-the-container-logs)
+  - [Login into the running container to use **KubeStellar** from inside the container](#login-into-the-running-container-to-use-kubestellar-from-inside-the-container)
+  - [Teardown and cleanup](#teardown-and-cleanup)
+
+## Install the pre-requisites
+
+Install `nake`:
+
+```bash
+sudo apt install -y make
+```
+
+Install `docker`:
+
+```bash
+sudo mkdir -p /etc/apt/keyrings
+curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo gpg --dearmor -o /etc/apt/keyrings/docker.gpg
+echo "deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/keyrings/docker.gpg] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable" | sudo tee /etc/apt/sources.list.d/docker.list > /dev/null
+sudo apt update
+sudo apt install -y docker-ce docker-ce-cli containerd.io docker-compose-plugin
+sudo usermod -aG docker $USER
+```
+
+Or `podman` (multi-arch `buildx` option will not be available):
+
+```bash
+sudo apt install -y podman
+alias docker=podman
+```
+
+## Build a local container image
+
+Build a local container image `kubestellar` for the host architecture using the latest release of **KubeStellar** with the command:
+
+```bash
+make build
+```
+
+A desired version of **KubeStellar** can be specified by using the optional `KUBESTELLAR_VERSION` argument, such as `KUBESTELLAR_VERSION=v0.3.1`.
+
+The **KubeStellar** version is used to tag the container image, as shown below:

```

Comment:
This wording leaves me uncertain whether setting the KUBESTELLAR_VERSION variable only controls the image tag or whether it also controls version of KubeStellar that goes into the image.  I think the truth is that it does both.  I think it would be better to introduce this variable with a sentence that says that it does both.

Reply:
ok reworded